packages/calling-component-bindings/src/utils/videoGalleryUtils.ts

```diff
@@ -103,3 +115,18 @@ const convertRemoteVideoStreamToVideoGalleryStream = (stream: RemoteVideoStreamS
     renderElement: stream.view?.target
   };
 };
+
+const convertRemoteParticipantStateToVideoGalleryRemoteParticipantState = (
+  state: RemoteParticipantConnectionState
+): VideoGalleryRemoteParticipantState | undefined => {
+  if (state === 'Idle' || state === 'Connecting') {
+    return 'Connecting';
+  }
+  if (state === 'EarlyMedia' || state === 'Ringing') {
```

Comment:
What is `EarlyMedia`, and how does it differ from `Ringing`?

Should the logic for how to deal with `EarlyMedia` be in the UI component instead of here?
Justification: We avoid diff from headless base types unless we have a reason for it (in that case add a comment to the code above about why we are mapping these states together)